tests/test_blank_frame_logging.py

```
@@ -0,0 +1,39 @@
+
+            with (
+                patch("app.utils.scheduling.SCREENSHOT_DIRECTORY", tmp),
+                patch("app.utils.scheduling.capture_or_download", return_value=True),
+                patch("app.utils.scheduling.get_template", return_value=template),
+                patch("app.utils.scheduling.is_mostly_blank", return_value=True),
+                patch("app.utils.scheduling.os.symlink"),
+                patch("app.utils.scheduling.os.rename"),
+                patch("app.utils.scheduling.os.unlink"),
+                patch("app.utils.scheduling.get_cached_status_code", return_value=404),
+            ), self.assertLogs(level="INFO") as logs:
+                scheduling.update_camera("cam1", template)
```

Pylint (3.3.7)

[error] Context manager 'tuple' doesn't implement __enter__ and __exit__.

(E1129)